packages/sku/lib/program/commands/init/init.action.js

```diff
@@ -228,7 +203,8 @@ const packageNameRegex =
   );
   console.log();
 
-  const logLevel = args.argv.indexOf('--verbose') >= 0 ? 'verbose' : 'regular';
+  // TODO: add loglevel to sku init as option.
```

Comment:
Is this a TODO for this PR?

If it's just a suggestion for a later change I'd suggest leaving the comment out and tracking the suggested change/feature separately.

Reply:
Tying this in to whatever you said in your general comment. How about I re-work the `-d, --debug` flag to actually accept a level and we use that to determine all the other levels of logging in the application?

I don't think I can get everything to the correct level of logging throughout the application in one go. But doing this sets the CLI api so that we can upgrade things over time that make more sense.

Reply:
How do you feel about just removing the comment?
Or change the comment to describe why it is how it is rather than saying how it should change in the future.

Reply:
I feel good removing the comment. I'm just spitballing here on how we want to do all the error handling. I left it in there for me to have a second look at this. What I'm going to do is update the description of the option in the command interface and remove this todo.